packages/config/src/Config.types.ts

```diff
@@ -520,6 +520,14 @@ export type WebPlatformConfig = {
    * Configuration for PWA splash screens.
    */
   splash?: WebSplashScreen;
+  config?: {
+    /**
+     * [Firebase Configuration Object](https://support.google.com/firebase/answer/7015592) Firebase web configuration.
+     */
+    firebase?: {
```

Comment:
Perhaps `googleServicesFile` so it matches iOS and Android?

Reply:
I initially considered that as well, but was having trouble parsing the file through the webpack plugin. My code wasn't being called in places where I expected it.  And in `createEnvironmentConstants` I couldn't access (or don't know how to safely) the project-root.

Do you have any recommendations for me?